test/parallel/test-http-agent-timeout.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+{
+  // Ensure reuse
+
+  const agent = new http.Agent({ keepAlive: true });
+
+  const server = http.createServer(function(req, res) {
```

Comment:
Nit: any particular reason to use `function` instead of `=>` in these tests, or just a copy-paste of another test?

Reply:
> copy-paste of another test